src/Drive.php

```diff
@@ -304,7 +322,8 @@ public function getResources(string $path = "/"): array
             throw ExceptionHelper::getHttpErrorException($e);
         }
 
-        return $resources;
+        // make sure there is again an element with index 0
```

Comment:
should we also put a comment for line 320 (unset) to mention why that is done - I think that is removing the entry for a folder itself.

Reply:
done